interactions:
- request:
    body: '{"biller_id": 40, "account_number": "501000000004"}'
    headers:
      Accept: [application/vnd.regalii.v3.1+json]
      Accept-Encoding: ['gzip, deflate']
      Authorization: ['APIAuth 88879c1b066dc9aea6201f27be2bbba9:LDrtakmWkvZswhhs+Po9hj/5guU=']
      Connection: [keep-alive]
      Content-Length: ['51']
      Content-MD5: [mDRNhwgEiKn41ooFxmHjFQ==]
      Content-Type: [application/json]
      Date: ['Sat, 08 Dec 2018 02:14:33 GMT']
      User-Agent: [python-requests/2.20.1]
    method: POST
    uri: https://api.casiregalii.com/bills
  response:
    body: {string: '{"code":"R2","message":"Invalid Account Number","id":8389}'}
    headers:
      Cache-Control: [no-cache]
      Connection: [keep-alive]
      Content-Type: [application/json; charset=utf-8]
      Date: ['Sat, 08 Dec 2018 02:14:34 GMT']
      Strict-Transport-Security: [max-age=31536000]
      Transfer-Encoding: [chunked]
      X-Content-Type-Options: [nosniff]
      X-Frame-Options: [SAMEORIGIN]
      X-Request-Id: [dc710cb1-3a52-48c6-bcf3-f8a811cdff11]
      X-Runtime: ['1.057535']
      X-XSS-Protection: [1; mode=block]
    status: {code: 422, message: Unprocessable Entity}
version: 1
